src/wallet/wallet.cpp

```diff
@@ -1650,6 +1699,59 @@ void CWallet::AvailableCoins(vector<COutput>& vCoins, bool fOnlyConfirmed, const
 
     {
         LOCK2(cs_main, cs_wallet);
+#if 1
```

Comment:
I think you should remove this directive and the unused code for better reading of the effective changes and prevent code pollution.